src/assets/themes/gothamist/variables.scss

```diff
@@ -42,37 +42,25 @@ $breakpoints: (
     "max": $max,
 );
 
-//colors
-$colors: (
-    "blue": #0454d6,
-    "lightblue": #90d8fc,
-    "redorange": #ff3500,
-    "orange": #ff6a00,
-    "lightorange": #f79800,
-    "gold": #f5b80c,
-    "pink": #ffb5e0,
-    "lightpink": #ffdfdf,
-    /*     "green": #7fa366,
-    "yellow": #dac57a,
-    "cyan": #71abc2,
-    "pink": #bf616a,
-    "indigo": #4e6aa4,
-    "teal": #5eac90,
-    "orange": #d08770,
-    "bluegray": #7e8d9f,
-    "purple": #9a6796, */
-);
-
 // scale-color example
 // $newColor: scale-color($existingColor, $saturation: 30%, $lightness: -30%, $alpha: -90%) !default;
 // can use any of the following $red, $green, $blue, $saturation, $lightness, $whiteness, $blackness, $alpha
 
-$primaryColor: #f4f2f0 !default;
-$primaryColorAlt: scale-color($primaryColor, $lightness: 15%) !default;
+// colors
+// add these to the cssvars file as well
+// if you need to use these colors later in this file, use map-get($colors, "red")
+$colors: (
+    "soybean": #D8D7AF,
+    "lightsoybean": #F3F3E5,
+    "red": #DE1E3D
+);
+
+$primaryColor: #111111 !default;
```

Comment:
looks like you use #111111 a lot. Add it to the colors array and map-get() it everywhere you use it.

Reply:
It wouldn't let me do this - getting an error:

SassError: $color: "#111111" is not a color.
61 │ $primaryLightColor: scale-color($primaryColor, $lightness: 30%) !default;